cpg-neo4j/src/main/kotlin/de/fraunhofer/aisec/cpg_vis_neo4j/Application.kt

```diff
@@ -232,6 +253,94 @@ class Application : Callable<Int> {
     private val passList: List<String>
         get() = passClassList.mapNotNull { it.simpleName }
 
+    private val packages: Array<String> =
+        arrayOf("de.fraunhofer.aisec.cpg.graph", "de.fraunhofer.aisec.cpg.frontends")
+
+    fun serializeCpgViaOGM(
```

Comment:
I would document this function for better maintainability. Also I would rename some function as I was confused and that might happen to others IMO. Here `translateCPGToOGMBuilders`?

Reply:
done